blogs/whats_new_1.6/index.md

```diff
@@ -0,0 +1,231 @@
+
+
+## Graphs and Logs Without Loki and Storage
+
+Prior to 1.4, NetObserv required an external component called Loki as well as storage, such as an S3 bucket, to store logs.  These flow logs allowed NetObserv to provide a rich UI to display graphs, tables, and a topology.  The resources required by Loki can be significant, particularly if you have lots of traffic and are sampling all data, not to mention the storage required.  In 1.4 and 1.5, if you have your own observability platform and only need the flow logs data, you can simply export this data and not install Loki or provide storage.  When you do this, you essentially get no UI besides some minimal flow metrics in **Observe > Dashboards**, because the expectation is that your platform will provide the features and the visuals.
+
+This release changes that premise.  It brings back the graphs in the **Overview** panel and the topology in the **Topology** panel.  This is huge because the core of Network Observability is now fully functional at a fraction of the resources required when Loki is used!  It achieves this by creating Prometheus metrics from the flows and storing them at 15-second intervals.
+
+So what's the catch?  Without storing the flow logs as JSON data, there will be some impact.  The most notable is that there won't be a traffic flows table, because flows are no longer stored (Figure 1).
+
+![Traffic flows grayed out](images/traffic_flows-grayed_out.png)
+_<div style="text-align: center">Figure 1: Traffic flows grayed out</div>_
+
+The other point to understand is that the metrics don't have information at the pod level so for example, in the topology, the **Resource** scope, which shows the pod to pod/service communication, will not exist (Figure 2).
+
+![Topology](images/topology-no_resources.png)
+_<div style="text-align: center">Figure 2: Topology - No "Resources"</div>_
+
+A couple of other features, namely packet drop reason and multi-tenancy, are not supported but will be addressed in a follow on release.  If you need any of these capabilities, then go ahead and install Loki and provide storage as usual.
+
+Let's walk through how to configure a Loki-less setup.  Install the Network Observability Operator.  In **Operators > Installed Operators**, click the **Flow Collector** link, and then **Create FlowCollector**.  Click **Loki client settings** to open this up (Figure 3).
+
+![Loki client settings](images/flp-loki_client_settings.png)
+_<div style="text-align: center">Figure 3: Loki client settings</div>_
+
+By default, Loki is enabled so set **Enable** to false.  That's it!
+
+There is one other note worth mentioning.  Even if you do install Loki, by default, it will favor using the metrics instead of Loki for querying whenever possible.  By doing this, not only will it be faster, but it will be possible to query data over a period of weeks or months.  This behavior can be changed under **Prometheus**, **Querier** in the **Enable** setting (Figure 4).
```

Comment:
```suggestion
There is one other note worth mentioning.  Even if you do install Loki, by default, the Network Observability Operator favors using the metrics instead of Loki for querying whenever possible.  By doing this, not only is the Operator experience faster, but it is also possible to query data over a period of weeks or months.  This behavior is configurable in the **Prometheus**, **Querier** in the **Enable** setting (Figure 4).
```

Reply:
A few suggestions to update the future tense to present tense. Additionally, I think "it" needs to be clarified to "Network Observability Operator".

Reply:
It's not the Network Observability Operator that's doing this; it's technically the NetObserv console plugin, but I rather not say that.  The point is that in the Overview panel and Topology, it favors using metrics instead of Loki.

Reply:
Understood, and that makes sense. Thank you!